src/makeInfinityAdder.js

```diff
@@ -1,10 +1,24 @@
 'use strict';
 
-/***
- * @return {function}
- */
+// /***
+//  * @return {function}
+//  */
+
 function makeInfinityAdder() {
-  // write code here
+  let count = 0;
+
+  return function countFunction(a) {
+    if (a === undefined) {
```

Comment:
```suggestion
    if (!value) {
```

Reply:
hello, I can not do it because, sometimes value = 0.
![image](https://github.com/user-attachments/assets/272a3773-e42a-4f14-989b-951026c4df95)